dev/ci/ci-bedrock2.sh

```diff
@@ -6,4 +6,4 @@ ci_dir="$(dirname "$0")"
 FORCE_GIT=1
 git_download bedrock2
 
-( cd "${CI_BUILD_DIR}/bedrock2" && git submodule update --init --recursive && COQMF_ARGS='-arg "-async-proofs-tac-j 1"' make )
+( cd "${CI_BUILD_DIR}/bedrock2" && git submodule update --init --recursive && COQMF_ARGS='-arg "-async-proofs-tac-j 1"' make | env -i LANG=en_US.UTF-8 grep -av '[^[:print:]]' `#9767` )
```

Comment:
This grep doesn't work for my local testing, OTOH piping through `iconv -t UTF-8 -c` works (we already have iconv in the image)
Found through https://unix.stackexchange.com/a/6517